Minor grammar fixes (#728)
graphql/graphql-spec: spec/Section 6 -- Execution.md

**Merging Selection Sets**

> When multiple fields of the same name are executed in parallel, their

Reviewer: “More than one” is more explicit here; instead change fields to field and are to is?

Reviewer 2: As comically fussy as it I'm being here, the problem with the phrase "more than one" is that it causes an awkward tension between the singular and plural forms used in the sentence: I _could_ revert "multiple" to "more than one", and change "fields" to "field", but I don't think I could change "are" to "is" because then I would have to change "their" to "its" which would change the meaning. The sentence would then read,

> When more than one field of the same type are executed in parallel, their selection sets are merged together...

Would this work for you?

Reviewer: I'm not sure that's quite right; the following are all gramatically correct, I believe:

- "When a field is executed, its ..."
- "When two fields are executed, their ..."
- "When more than one field is executed, their ..."

The "is" applies to "field" (singular), whereas the "their" applies to "more than one field" (plural). Yes, English is weird.

> When a noun phrase contains more than one and a singular noun, the verb is normally singular: More than one editor is working on that project.
> -- https://www.ahdictionary.com/word/search.html?q=more

Reviewer 2: Ah! What a fascinating rule: in that case you're right and I retract my objection :)

Reviewer: `multiple` is less precise than `more than one`, so I'd instead go with this:

```suggestion
When more than one field of the same name is executed in parallel, their
```
